example/gardener-local/gardenlet/values-kind-ha-mz.yaml

```diff
@@ -0,0 +1,8 @@
+global:
```

Comment:
Why is this still a separate file? You can put the content into `values-kind-ha.yaml` as well.